src/ServiceManagerBase.sol

```diff
@@ -105,24 +111,34 @@ abstract contract ServiceManagerBase is OwnableUpgradeable, ServiceManagerBaseSt
             for (uint256 j = 0; j < numStrategies; ++j) {
                 IStrategy strategy = _stakeRegistry.strategyParamsByIndex(i, j).strategy;
                 strategies[j] = strategy;
+                require(
```

Comment:
can we just not check this at all and have the call to OSM revert?

Reply:
also works assuming it will revert